src/frontend/components/UI/DialogHandler/components/MessageBoxModal/index.tsx

```diff
@@ -17,8 +17,24 @@ interface MessageBoxModalProps {
   type: DialogType
 }
 
+function decodeHTML(html: string) {
+  const txt = document.createElement('textarea')
+  txt.innerHTML = html
+  return txt.value.split('\n').map((item, key) => {
+    return (
+      <span key={key}>
+        {item}
+        <p />
+      </span>
+    )
+  })
+}
+
 const MessageBoxModal: React.FC<MessageBoxModalProps> = function (props) {
   const { t } = useTranslation()
+
+  const message = decodeHTML(props.message)
```

Comment:
maybe too performance nit picky, but this can probably be memoized so it doesn't run on every render and only of porps.message changes